bigchaindb/backend/localmongodb/schema.py

```diff
@@ -142,8 +143,14 @@ def create_abci_chains_indexes(conn, dbname):
     conn.conn[dbname]['abci_chains'].create_index('height',
                                                   name='height',
                                                   unique=True,)
-
-    logger.info('Create `abci_chains.chain_id` secondary index.')
```

Comment:
There is a conflict resolution leftover.

Reply:
Restored